estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Real Estate',
+    'version': '1.0',
+    'description': """Real Estate module""",
+    'author': 'Vauxoo',
+    'license': 'GPL-3 or any later version',
+    'depends': ['base'],
+    'application': True,
+    'installable': True,
+    'maintainer': 'Suavesito-Olimpiada',
```

Comment:
Please delete key. In this exercise, the __manifest__.py file should only define the name and the dependencies of our modules. The only necessary framework module for now is base.

Reply:
I'll clean it to the minimum.